openebs/openebs-provisioner.go

```diff
@@ -151,7 +152,10 @@ func (p *openEBSProvisioner) Delete(volume *v1.PersistentVolume) error {
 	}
 
 	// Issue a delete request to Maya API Server
-	openebsVol.DeleteVsm(volume.Name)
+	err := openebsVol.DeleteVsm(volume.Name)
+	if err != nil {
+		return err
```

Comment:
May be you could format the error and send it, so that its clear that it failed on delete volume.